pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -157,6 +157,10 @@ const (
 	createTagInitialDelay = 1 * time.Second
 	createTagFactor       = 2.0
 	createTagSteps        = 9
+
+	// Number of node names that can be added to a filter. The AWS limit is 200
+	// but we are using a lower limit on purpose
```

Comment:
Sounds good.  If nothing else, it leaves us room for our own additional filters.